package.json

```diff
@@ -7,7 +7,7 @@
     "url": "git://github.com/mxstbr/react-boilerplate.git"
   },
   "engines": {
-    "npm": ">=3"
+    "npm": ">=5"
```

Comment:
npm is still version 3 with node 5 and upwards, I think you meant node >= 5 here.